common/src/main/java/com/redhat/cloud/notifications/transformers/SeverityTransformer.java

```diff
@@ -60,117 +76,99 @@ public Severity getSeverity(Action action) {
      *
      * @return The highest severity within the payload events.
      */
-    private Severity extractLegacySeverity(final Action action) {
+    private Optional<Severity> extractLegacySeverity(final Action action) {
         List<Event> events = action.getEvents();
-        if (events.isEmpty()) {
-            return Severity.UNDEFINED;
-        }
+        Optional<Severity> severity = Optional.empty();
+        if (!events.isEmpty()) {
 
-        ArrayList<Severity> severities = switch (action.getApplication()) {
-            // Both OpenShift Advisor and RHEL Advisor use the same layout here
-            case "advisor" -> new ArrayList<>(events.stream().map(
+            Stream<Severity> severities = switch (action.getApplication()) {
```

Comment:
`inventory` is gone, is that intentional?

Reply:
Yes because now its values are driven by DB  event types default values